c10/util/numa.cpp

```diff
@@ -0,0 +1,137 @@
+#include "c10/util/numa.h"
+
+#if USE_NUMA && defined(__linux__) && !defined(C10_DISABLE_NUMA) && C10_MOBILE == 0
```

Comment:
here I added `USE_NUMA`

Reply:
how exactly does it work? C++ includes and cmake variables are two totally different namespaces

Reply:
You should update c10/macros/cmake_macros.h.in to get C10_USE_NUMA to get passed through instead of leaving it in caffe2.